Bottom: 4acd7139c3a44b1db418e1578c66cf1dbd7fa654
Top:    ba6b8c94244b8c6ce426b33fa7f799ad9cf0ea8d
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-02 16:00:37 +0800

Refresh of vhost-part

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index 6947425..c0cbc21 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -556,13 +556,13 @@ static netdev_tx_t macvlan_start_xmit(struct sk_buff *skb,
 	unsigned int len = skb->len;
 	int ret;
 
+	printk("start xmit!\n");
+
 	if (unlikely(netpoll_tx_running(dev)))
 		return macvlan_netpoll_send_skb(vlan, skb);
 
 	ret = macvlan_queue_xmit(skb, dev);
 
-	printk("here!\n");
-
 	if (likely(ret == NET_XMIT_SUCCESS || ret == NET_XMIT_CN)) {
 		struct vlan_pcpu_stats *pcpu_stats;
